data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/MessageFormat.java

```diff
@@ -17,7 +17,7 @@
  */
 public enum MessageFormat {
 
-    PLAINTEXT("plaintext"), JSON("json"), AVRO("avro"), BYTES("bytes");
+    PLAINTEXT("plaintext"), JSON("json"), AVRO("avro"), BYTES("bytes"), JSON_OTEL_TRACE("json_otel_trace");
```

Comment:
Using `otel_trace_json` would more closely align with the input codec naming convention. 

In that scenario, the code is `otel_logs` and a configure yields `json` or `protobuf`. So the `otel_logs` comes first.